workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -70,25 +75,95 @@ private static void LaunchMobileClient()
                     .WithArgs("shell", "am", "start", "-S", "-n", $"{bundleId}/com.unity3d.player.UnityPlayerActivity",
                         "-e", "\"arguments\"", $"\\\"{arguments.ToString()}\\\"")
                     .Run();
-
-                EditorUtility.DisplayProgressBar("Launching Mobile Client", "Done", 1.0f);
             }
             finally
             {
                 EditorUtility.ClearProgressBar();
             }
         }
 
-        private static bool TryGetApkPath(string rootPath, out string apkPath)
+        [MenuItem(MenuLaunchiOSDevice, false, 74)]
+        private static void LaunchiOSDeviceClient()
         {
-            foreach (var file in Directory.GetFiles(rootPath, "*.apk", SearchOption.AllDirectories))
+            try
             {
-                apkPath = file;
-                return true;
-            }
+                // Ensure needed tools are installed
+                if (string.IsNullOrEmpty(LibIDeviceInstallerBinary))
+                {
+                    Debug.LogError("Could not find ideviceinstaller tool. Please ensure it is installed. " +
+                        "See https://github.com/libimobiledevice/ideviceinstaller fore more details.");
```

Comment:
typo:
```suggestion
                        "See https://github.com/libimobiledevice/ideviceinstaller for more details.");
```

After testing this pr and going to this page, I'm actually more confused. Do I actually need to clone their repo and build it?
I simply used `brew` to install it, which seems to work. but we also don't specify which version of it we actually need / have tested against